### PR TITLE
[skip ci] ceph_key: add fetch_initial_keys capability

### PR DESCRIPTION
**Pull request by leseb**
This is needed for Nautilus since the ceph-create-keys script goes away.
(https://github.com/ceph/ceph/pull/21305)
Now the module if called with 'state: fetch_initial_keys' will lookup
keys generated by the monitor and write them down on the filesystem to
the right location (/etc/ceph and /var/lib/ceph/boostrap*).
This is not applicable to container since keys are generated by the
container only.

Signed-off-by: Sébastien Han <seb@redhat.com>

### CODE REVIEW

**Comment by leseb** [comment]
TODO: 

- [ ] finish up the tests

**Comment by leseb** [comment]
jenkins test dev-centos7_cluster

**Comment by leseb** [comment]
jenkins test dev-docker_cluster

**Comment by leseb** [comment]
jenkins test dev-centos7_cluster

**Comment by leseb** [comment]
jenkins test dev-centos7_cluster

**Comment by leseb** [comment]
jenkins test dev-centos7_cluster

**Comment by leseb** [comment]
jenkins test pipeline

File: roles/ceph-mon/tasks/ceph_keys.yml [hunk 1]
```diff
@@ -1,9 +1,28 @@
 ---
+- name: waiting for the monitor(s) to form the quorum...
+  command: "ceph --cluster {{ cluster }} -n mon. -k /var/lib/ceph/mon/{{ cluster }}-{{ ansible_hostname }}/keyring mon_status --format json"
+  register: ceph_health_raw
+  until: >
+    (ceph_health_raw.stdout | from_json)['state'] in ['leader', 'peon']
```
Comment (guits) [inline roles/ceph-mon/tasks/ceph_keys.yml]: `(ceph_health_raw.stdout | from_json)['state'] not in ['leader', 'peon']` ?

Comment (guits) [inline roles/ceph-mon/tasks/ceph_keys.yml]:
```suggestion
    (ceph_health_raw.stdout | from_json)['state'] not in ['leader', 'peon']
```

**Comment by leseb** [comment]
jenkins test dev-centos7_cluster

**Comment by leseb** [comment]
Needs https://github.com/ceph/ceph-container/pull/1237/

**Pull request merged**